modules/standard/Sort.chpl

```diff
@@ -377,25 +391,65 @@ proc chpl_check_comparator(comparator, type eltType) param {
     }
   }
   else if canResolveMethod(comparator, "compare", data, data) {
+    // TODO: check for keyPartComparator or relativeComparator
```

Comment:
Did you mean to say "keyComparator" instead of "keyPartComparator" here?

Reply:
No, `.compare` can be called on a `keyPartComparator` if radix sort is not appropriate for whatever reason.

But I think to write this logic properly we need a `relativeComparator` to also check against. 

```
if not implementsKeyPart and not ImplementsRelative then
  ...deprecation warning...
```